### PR TITLE
DOC-255-Cloud-Upgrades-Maintenance-Windows

### PR DESCRIPTION
**Pull request by micheleRP**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2467
Review deadline: Wed Sept 18

## Page previews
[Redpanda Overview - Maintenance windows](https://deploy-preview-68--rp-cloud.netlify.app/redpanda-cloud/get-started/cloud-overview/#maintenance-windows)

## Checks

- [X] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/get-started/pages/cloud-overview.adoc [hunk 1]
```diff
@@ -177,7 +177,11 @@ BYOC with customer-managed VPC::
 
 == Maintenance windows
 
-By default, Redpanda Cloud upgrades take place on Tuesdays. Optionally, on the **Cluster settings** page, you can select a window of specific off-hours for your business for Redpanda to apply updates. All times are in Coordinated Universal Time (UTC). Updates may start at any time during that window. 
+Redpanda runs maintenance operations on clusters in a rolling fashion, accompanied by a series of health checks, so there is no disruption to the availability of your service. As part of the Kafka protocol, recycling nodes cause client connections to be restarted. All mainstream client libraries support automatic reconnections for this.
+
+By default, Redpanda Cloud maintenance cycles start on Tuesdays. Optionally, on the **Cluster settings** page, you can schedule a day and window of time for your business for Redpanda to apply updates. All times are in Coordinated Universal Time (UTC), and operations typically take six hours. Maintenance may start at any time during that window, and some operations may complete after the window ends.
```
Comment (towfiqa) [inline modules/get-started/pages/cloud-overview.adoc:182]: Please replace with:
By _default_, Redpanda Cloud maintenance cycles start on Tuesdays and we may run operations on any day at any time. You are able to override the default to _Scheduled_ on the **Cluster settings** page. By choosing the day and start time which will limit Redpanda Cloud to run operations only the day and start hour specified. Maintenance windows are typically 6 hours in duration. All operations will begin during the maintenance window and some operations may complete after. All times are in Coordinated Universal Time (UTC).

**Review by towfiqa — APPROVED** [review]

File: modules/get-started/pages/cloud-overview.adoc [hunk 2]
```diff
@@ -177,7 +177,11 @@ BYOC with customer-managed VPC::
...
 
-By default, Redpanda Cloud upgrades take place on Tuesdays. Optionally, on the **Cluster settings** page, you can select a window of specific off-hours for your business for Redpanda to apply updates. All times are in Coordinated Universal Time (UTC). Updates may start at any time during that window. 
+Redpanda runs maintenance and upgrade operations on clusters in a rolling fashion, accompanied by a series of health checks, so there is no disruption to the availability of your service. As part of the Kafka protocol, recycling nodes cause client connections to be restarted. All mainstream client libraries support automatic reconnections for this.
+
+By default, Redpanda Cloud maintenance cycles start on Tuesdays and may run on any day at any time. You can override this default and schedule a specific maintenance window on the *Cluster settings* page. A *Scheduled* maintenance window requires Redpanda Cloud to run operations on the day and time specified. Maintenance windows typically take six hours. All operations begin during the maintenance window, but some operations may complete after the window ends. All times are in Coordinated Universal Time (UTC).
```
Comment (Feediver1) [inline modules/get-started/pages/cloud-overview.adoc:182]: This is confusing. How about:
By default, Redpanda Cloud maintenance cycles start on Tuesdays; however, they may run on any day at any time.

**Review by Feediver1 — APPROVED** [review]
See suggested clarification

File: modules/get-started/pages/cloud-overview.adoc (hunk 2 above)
Comment (micheleRP) [reply modules/get-started/pages/cloud-overview.adoc:182]: I think the first sentence is saying that Redpanda may do maintenance operations whenever we want (not specifically on 
Tuesdays: that's just when the cycle starts). @towfiqa is this edit correct/ok:

By default, Redpanda may run maintenance operations on any day at any time. You can override this default...
...
TIP: Redpanda Cloud maintenance cycles always start on Tuesdays. Clusters scheduled on Tuesdays are updated first, and clusters scheduled on Mondays are updated last. Keep this in my mind when sequencing updates for multiple clusters.

**Comment by micheleRP** [comment]
> See suggested clarification

Thanks, @Feediver1: Towfiqa approved my rewrite.

**Pull request merged**